src/components/Navigator/NavigatorProvider.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ReactNode, useState } from "react";
+
+import { NavigatorContext } from "./useNavigatorContext";
+
+export const NavigatorProvider = ({ children }: { children: ReactNode }) => {
```

Comment:
if you touch navigator components - don't you think to rename it? It is a search bar, perhaps we could rename it?

Reply:
it can be SearchOverlay, NavigatorSearch, NavigatorSearchBar or something similar?